ts/parsers/paragraphparser.ts

```diff
@@ -224,18 +232,27 @@ export default class ParagraphParser {
 
     /**a:rPr */
     public static determineTextProperties(textProperties): Content["textCharacterProperties"] {
+        const defaultProperties: Content["textCharacterProperties"] = {
+            size: 1200,
+            fontAttributes: [],
+            font: "Helvetica",
+            fillColor: "000000"
```

Comment:
should it not be `#000000`

Reply:
mm yea, but we are not using colors, and its been like that in the initial code, if i change i have to change it in many places. 
so skipping this change for now, if that's okay with you ?

Reply:
Yeah fair point, OK